index.js

```diff
@@ -8,18 +8,18 @@ function main(stream) {
 	var render = function () {
 		cliCursor.hide();
 		var out = [].join.call(arguments, ' ') + '\n';
-		stream.write(ansiEscapes.eraseLines(prevLineCount) + out);
+		render.clear();
+		stream.write(out);
```

Comment:
The original line cleared the previous log and printed the new log. The clearing bit was a duplication of what the clear function does, so I thought it might be better to just call it (keeping with the DRY principle).
The other reason to separate it, is that if clear will ever be broken again you will know it immediately since the output will be messed up.

That said, if you think there's a possible performance issue, I'll change it.